fapi-client/src/main/scala/com/gu/facia/api/utils/CardStyle.scala

```diff
@@ -20,7 +20,7 @@ object CardStyle {
   val news = "news"
 
   def apply(content: Content, trailMetaData: MetaDataCommonFields): CardStyle = {
-    val href = trailMetaData.href.orElse(content.safeFields.get("href"))
```

Comment:
I don't understand the code much, but I've noticed this change in few places, aren't we missing something by getting rid of `content.safeFields.get("href")`?